dataduct/database/database.py

```diff
@@ -61,6 +65,7 @@ def add_relation(self, relation):
 
         self._relations[relation.full_name] = relation
 
+    @property
```

Comment:
Oh on second thought, these don't have setters...

what do you think about changing this to 'get_relations'?